draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -694,20 +694,20 @@ RADIUS/UDP packets MUST NOT be sent to this port.
 RADIUS/DTLS clients SHOULD NOT probe servers to see if they support DTLS transport.
 Instead, clients SHOULD use DTLS as a transport layer only when administratively configured.
 
-## Session Management
+## Connection Management
 
-Where RADIUS/TLS can rely on the TCP state machine to perform session tracking, RADIUS/DTLS cannot.
-As a result, implementations of RADIUS/DTLS may need to perform session management of the DTLS session in the application layer.
+Where RADIUS/TLS can rely on the TCP state machine to perform connection tracking, RADIUS/DTLS cannot.
+As a result, RADIUS/DTLS implementations may need to perform DTLS connection tracking in the application layer.
 This subsection describes logically how this tracking is done.
 Implementations MAY choose to use the method described here, or another, equivalent method.
 When implementations do not use the 5-tuple described below, note that IP address based policies MUST still be applied for all incoming packets, similar to the mandated behavior for TLS Session Resumption in {{tls_session_resumption}}.
 
 We note that {{RFC5080, Section 2.2.2}}, already mandates a duplicate detection cache.
-The session tracking described below can be seen as an extension of that cache, where entries contain DTLS sessions instead of RADIUS/UDP packets.
+The connection tracking described below can be seen as an extension of that cache, where entries contain DTLS connections instead of RADIUS/UDP packets.
 
-### Server Session Management
+### Server Connection Management
```

Comment:
I'm not exactly sure if we don't actually mean Session in this instance. Just as a reminder for me, so I don't forget.